src/screens/onboarding/components/step-content/common.ts

```diff
@@ -1,7 +1,10 @@
+import { Asset } from 'react-native-image-picker';
+
 const defaultAddUserInfoPayload = {
   firstName: '',
   lastName: '',
   position: '',
+  avatar: undefined as Asset | undefined,
```

Comment:
Looks overcomplicated to me. Can we simplify type here?

Reply:
Ask @katyakostikova. I think no :)